docs(configuration): document resolve.roots (#3827)
webpack/webpack.js.org: src/content/configuration/resolve.md

};
```

### `resolve.roots`

`[string]`

> A list of directories where requests of server-relative URLs (starting with '/') are resolved. On non-windows system these requests are resolved as absolute path first.

Reviewer: Can we say `systems`? Its more common these days
```suggestion
A list of directories where requests of server-relative URLs (starting with '/') are resolved. On non-windows systems these requests are resolved as absolute path first.
```

> A list of directories where requests of server-relative URLs (starting with '/') are resolved, defaults to `[options.context]`. On non-windows systems these requests are resolved as absolute path first.

Reviewer 2: ```suggestion
A list of directories where requests of server-relative URLs (starting with '/') are resolved, defaults to `[options.context]`. On non-Windows systems these requests are resolved as an absolute path first.
```

> A list of directories where requests of server-relative URLs (starting with '/') are resolved, defaults to `[options.context]`. On non-Windows systems these requests are resolved as an absolute path first.

Reviewer: ```suggestion
A list of directories where requests of server-relative URLs (starting with '/') are resolved, defaults to [`context` configuration option](/configuration/entry-context/#context). On non-Windows systems these requests are resolved as an absolute path first.
```
